doozerlib/config.py

```diff
@@ -114,6 +115,51 @@ def _do_print(kind, output):
         _do_print("Images", data['images'])
         _do_print(" RPMS ", data['rpms'])
 
+    def config_gen_csv(self, keys, as_type=None, output=None):
+        """
+        Generate csv file or print it out to STDOUT
+        """
+
+        if keys is None:
+            print('No --keys specified, please make sure you have at least one key to generate')
+            return
+
+        # split --keys=a,b,c,d to list [a,b,c,d]
+        keys = [c.strip() for c in keys.split(',')]
+
+        image_list = None
+        if as_type == "image" and not self.runtime.rpms:
+            image_list = self.runtime.image_metas()
+        if as_type == "rpm" and not self.runtime.images:
+            image_list = self.runtime.rpm_metas()
+
+        if image_list is None:
+            print('Not correct --type specified (--type image or --type rpm). '
+                  'Or not consist with global options: --images/-i and --rpms/-r')
```

Comment:
Tiny nitpick:
```suggestion
                  'Or not consistent with global options: --images/-i and --rpms/-r')
```
Otherwise, LGTM!